src/fundus_murag/ui/state.py

```diff
@@ -27,19 +27,20 @@ class ModelPickerDialogState:
 
 
 def reset_app_state(state: AppState):
-    gemini = GeminiFundusAssistant(state.selected_model)
-    gemini.reset_chat_session()
+    state.available_models = pd.DataFrame()
+    state.available_models = merge_models()
```

Comment:
please rename the `merge_models` method to `get_available_models_df` and remove Line 30